app/mailers/app_mailer.rb

```diff
@@ -1,13 +1,18 @@
 class AppMailer < ActionMailer::Base
-  default from: 'info@myflix.com'
+  default from: 'info@peter-myflix.herokuapp.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.